src/components/UserProfile/UserSettings/UserNotificationSettings/index.tsx

```diff
@@ -32,6 +32,9 @@ const messages = defineMessages({
   localuser: 'Local User',
   toastSettingsSuccess: 'Settings successfully saved!',
   toastSettingsFailure: 'Something went wrong while saving settings.',
+  pgpKey: 'PGP Public Key',
+  pgpKeyTip:
+    'By setting this key, any emails sent to you will be encrypted with this key.',
```

Comment:
```suggestion
    'Encrypt email messages using <PgpLink>PGP</PgpLink>',
```

We could link to https://en.wikipedia.org/wiki/Pretty_Good_Privacy perhaps? 😃

Also, do we need to do any input validation on the PGP key to ensure that users who don't understand what it is don't enter random things and complain that their email notifications stopped working?